XLS-0033d-multi-purpose-tokens/README.md

```diff
@@ -67,11 +67,28 @@ The **`MPTokenIssuance`** object represents a single MPT issuance and holds data
 
 ##### 1.2.1.1.1. **`MPTokenIssuance`** Ledger Identifier
 
-The ID of a MPTokenIssuance object, a.k.a. `MPTokenIssuanceID`, is the result of SHA512-Half of the following values, concatenated in order:
+The key of a `MPTokenIssuance` object, is the result of SHA512-Half of the following values, concatenated in order:
 
 * The MPTokenIssuance space key (0x007E).
+* The transaction sequence number.
 * The AccountID of the issuer.
+
+
+The ID of a `MPTokenIssuance` object, a.k.a. `MPTokenIssuanceID`, is a 192-bit integer, concatenated in order:
+
 * The transaction sequence number.
+* The AccountID of the issuer.
+
+```
+┌──────────────────────────┐┌──────────────────────────┐
+│                          ││                          │
+│      Sequence            ││     Issuer AccountID     │
+│      (32 bits)           ││        (160 bits)        │
+│                          ││                          │
+└──────────────────────────┘└──────────────────────────┘
+```
+
+**Note: The `MPTokenIssuanceID` is what the user inputs for the interface. Internally, the ledger splits the `MPTokenIssuanceID` into two components: `sequence` and `issuer` address.**
```

Comment:
> **Note: The `MPTokenIssuanceID` is what the user inputs for the interface.

Let's clarify this a bit more. E.g., which interface are we referring to here? I think there's a difference between API "interfaces" vs. the contents of a signed transaction, and on this point these proposed changes are a bit ambiguous.